message.go

```diff
@@ -410,3 +413,27 @@ func UIDWithLang(id int, lang Lang) int {
 func (m *Message) Is(mt MessageType) bool {
 	return m.Type == mt
 }
+
+// Ack if message come from AMQP. Acknowledges message processed successfully
+func (m *Message) Ack() error {
+	if m.External && m.Delivery != nil {
+		m.Delivery.Ack(false)
+	}
+	return nil
```

Comment:
Shouldn't you return an err if we ACK on something that's not functional?

Reply:
Titally correct, did it on my first iteration and forgot to add it now. Thanks for the catch